components/Favorites/Favorites.js

```diff
@@ -0,0 +1,78 @@
+import Link from "next/link";
+import StyledList from "../CarsList/StyledList";
+import { StyledLikeButton } from "../FilterForm/FilteredCarsList";
+import Image from "next/image";
+import { StyledResultDiv } from "../FilterForm/FilteredCarsList";
+
+function Favorites({ cars, handleToggleFavorite }) {
+  return (
+    <>
+      {cars.filter((car) => car.isFavorite).length < 1 ? (
+        <StyledResultDiv>{"You didn`t like anything yet"}</StyledResultDiv>
+      ) : (
+        <StyledList>
+          {cars
+            .filter((car) => car.isFavorite)
+            .map((car) => (
+              <li key={car.id}>
+                <StyledLikeButton onClick={() => handleToggleFavorite(car.id)}>
+                  {car.isFavorite ? (
+                    <Image
+                      src={"/heart.png"}
+                      alt="liked"
+                      width={40}
+                      height={40}
+                    ></Image>
+                  ) : (
+                    <Image
+                      src={"/notLiked.png"}
+                      alt="notLiked"
+                      width={40}
+                      height={40}
+                    ></Image>
+                  )}
+                </StyledLikeButton>
+                <Link href={`/detailsCar/${car.id}`}>
+                  <Image
+                    style={{ borderRadius: "10%" }}
```

Comment:
We do not want to work with inline stylings. Please use styled components instead.